Source/Shared/Extensions/JSON+Cache.swift

```diff
@@ -1,10 +1,19 @@
 import Foundation
 
+/// A configuration struct
+public struct CacheJSONOptions {
```

Comment:
@zenangst it is OK 
Although every "global" is actually under the framework namespace 😬